### PR TITLE
User Interface III - Austin Walela

### PR DESCRIPTION
**Pull request by walela**
test review comment

### CODE REVIEW

**Comment by walela** [comment]
submitted first final version. Please review and give feedback.

**Review by josenriagu — COMMENTED** [review]
Hey Austin,
I am unable to preview your work. However, you do not worry, We will have a look at it together during 1;1 today.

Cheers!!

File: great-idea/css/index.css [hunk 1]
```diff
@@ -63,4 +63,82 @@ h1, h2, h3, h4, h5 {
...
+	text-decoration: none;
+	color: grey;
+	padding: 20px;
+}
+.subheader {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items:flex-start;
+	align-content: space-between;
```
Comment (josenriagu) [inline great-idea/css/index.css:97]: You probably are trying to leave some spaces between these two elements in the **subheader** div. To make it work you need just;

**justify-content: space-between**
and
**align-items: center**

please fix!

**Review by josenriagu — CHANGES_REQUESTED** [review]
Great job Austin!

You are almost there! At the same time;
• always keep a close eye on the design document. You have some elements spilling over the border. check here: https://prnt.sc/opui0h

Cheers!! keep it up

**Pull request merged**